docs/source/commandline/commandline-all.rst

```diff
@@ -520,87 +519,87 @@ Core State/Playback Options
 
 **-record** / **-rec** *<filename>*
 
-	Specifies a file to record all input from a session. This can be used to record a  session for later playback. This feature does not work reliably for all systems, but can be used to watch a previously recorded session from start to finish. In order to make things consistent, you should only record and playback with all configuration (.cfg), NVRAM (.nv), and memory card files deleted. The default is NULL (no recording).
+	Specifies a file to record all input from a session. This can be used to record a session for later playback. This feature does not work reliably for all systems, but can be used to watch a previously recorded session from start to finish. In order to make things consistent, you should only record and playback with all configuration (.cfg), NVRAM (.nv), and memory card files deleted. The default is ``NULL`` (no recording).
 
 .. _mame-commandline-recordtimecode:
 
 **-record_timecode**
 
-	Tells MAME to create a timecode file. It contains a line with elapsed times on each press of timecode shortcut key (*default is F12*). This option works only when recording mode is enabled (**-record** option). The file is saved in the *inp* folder. By default, no timecode file is saved.
+	Tells MAME to create a timecode file. It contains a line with elapsed times on each press of timecode shortcut key (default is ``F12``). This option works only when recording mode is enabled (**-record** option). The file is saved in the *inp* folder. By default, no timecode file is saved.
```

Comment:
"F12" is a key label, and there's also a path fragment "inp" that hasn't been updated here.